src/components/incentives/MeritIncentivesTooltipContent.tsx

```diff
@@ -304,12 +320,44 @@ export const MeritIncentivesTooltipContent = ({
               />
             </Row>
           )}
+
+          {campaignConfig.type === CampaignType.STANDARD && (
+            <Row
+              height={24}
+              caption={
+                <Box sx={{ display: 'flex', alignItems: 'center' }}>
+                  {isMultipleCampaigns(meritIncentives.activeActions) ? (
+                    <Typography variant={typographyVariant}>Merit Incentives Combined</Typography>
+                  ) : (
+                    <Typography variant={typographyVariant}>Merit Incentives Standard</Typography>
```

Comment:
For default lets just use merit incentives

```suggestion
                    <Typography variant={typographyVariant}>Merit Incentives</Typography>
```